src/ovp_pipeline/source_authority.py

```
@@ -0,0 +1,261 @@
+
+from __future__ import annotations
+
+import json
+import logging
+import math
+import sqlite3
+from dataclasses import asdict
+from math import isfinite
+from datetime import datetime, timezone
+from pathlib import Path
+from typing import Any, Iterable
```

Ruff (0.15.12)

[warning] Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)